docs/integrations/sources/typeform.md

```diff
@@ -19,7 +32,7 @@ To get the API token for your application follow this [steps](https://developer.
 * In the left menu, click Personal tokens.
 * Click Generate a new token.
 * In the Token name field, type a name for the token to help you identify it.
-* Choose needed scopes \(API actions this token can perform - or permissions it has\). See here for more details on scopes.
+* Choose needed scopes \(API actions this token can perform - or permissions it has\). See [here](https://www.typeform.com/developers/get-started/scopes/) for more details on scopes.
```

Comment:
this step should only be shown in OSS not cloud

Reply:
moved this section to only be shown in OSS